oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -373,7 +375,7 @@ private static String getRpSid(AuthenticationRequest authRequest) {
     private List<VectorOfTrust> getVtrList(
             boolean reauthRequested, AuthenticationRequest authRequest)
             throws java.text.ParseException {
-        if (reauthRequested && orchestrationAuthorizationService.getVtrList(authRequest) == null) {
+        if (reauthRequested && isNull(authRequest.getCustomParameter(VTR_PARAM))) {
```

Comment:
Why do you prefer `isNull`, out of interest?

Reply:
I don't have a strong preference. I've seen both and try to be consistent. I guess maybe I gravitated towards it because C# which I'm more familiar with has the syntax `some_var is null`. I'm going to assume you have a different preference give you asked so will use == from now on.